pkg/cmd/tenant/variables/list/list.go

```diff
@@ -99,67 +102,169 @@ func listRun(cmd *cobra.Command, f factory.Factory, id string) error {
 		return err
 	}
 
-	vars, err := client.Tenants.GetVariables(tenant)
-	if err != nil {
-		return err
-	}
+	toggleValue, err := featuretoggle.IsToggleEnabled(client, "CommonVariableScopingFeatureToggle")
 
-	missingVariablesResponse, err := client.Tenants.GetMissingVariables(variables.MissingVariablesQuery{TenantID: vars.TenantID})
-	var missingVariables []variables.MissingVariable
-	if len(*missingVariablesResponse) > 0 {
-		missingVariables = (*missingVariablesResponse)[0].MissingVariables
-	}
-	var allVariableValues []*VariableValue
-	for _, element := range vars.LibraryVariables {
+	if toggleValue {
```

Comment:
I was going to recommend splitting this into two functions, one for toggle on and off. But perhaps this makes more sense if we are going to remove on half anyway.